src/main/java/com/cbfacademy/FileHandler/JSONFileHandler.java

```diff
@@ -1,14 +1,27 @@
-package main.java.com.cbfacademy.FileHandler;
+package com.cbfacademy.FileHandler;
+
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 
 import com.cbfacademy.Employee.Employee;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
 
 public class JSONFileHandler {
     
-    public List<Employee> readFile(String filename) {
-        
+    public static List<Employee> readFile(String filename) {
+        List<Employee> employees = new ArrayList<Employee>();
+        return employees;
     }
 
-    public void save(Employee employee, String outputFile) {
-
+    public static void save(Employee employee, String outputFile) {
+        Gson gson = new GsonBuilder().setPrettyPrinting().create();
+        try (FileWriter writer = new FileWriter(outputFile)) {
+            gson.toJson(employee, writer);
+        } catch (IOException e) {
+            e.printStackTrace();
+        }
```

Comment:
This is a good start. This will save the Employee given as JSON data in the given file. 

If there is data already in that file, it will overwrite it rather than add to it.